player_pyano/app.py

```diff
@@ -9,7 +9,7 @@
 LOCK = False
 SONGS_LIST = os.listdir("songs")
 
-output_name = mido.get_output_names()[0]
```

Comment:
how do you know this is the correct index?

Reply:
Not actually sure that the index here needed to change from 0 to 1 at all...